.github/workflows/deploy.yml

```
@@ -21,7 +21,9 @@ jobs:
             - name: Set up .NET Core
               uses: actions/setup-dotnet@v4
               with:
-                  dotnet-version: "8.0.x"
+				dotnet-version: |
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 24: found character that cannot start any token

(syntax-check)